# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sane-backends/hotfix-basename.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- sane-backends-1.2.1/frontend/scanimage.c.vanilla	2024-07-25 01:05:46.452880204 +0200
+++ sane-backends-1.2.1/frontend/scanimage.c	2024-07-25 01:11:56.724850161 +0200
@@ -40,9 +40,7 @@
 #include <unistd.h>
 #include <stdarg.h>
 
-#ifdef __FreeBSD__
 #include <libgen.h>
-#endif
 
 #if defined (__APPLE__) && defined (__MACH__)
 #include <libgen.h>     // for basename()
